reproschema/tests/test_rs2redcap_redcap2rs.py

```diff
@@ -451,5 +464,10 @@ def test_rs2redcap_redcap2rs(tmpdir):
         prot_tree_orig, prot_tree_final
     )
 
-    assert not errors_list, f"Errors: {errors_list}"
-    print("No errors, but found warnings: ", warnings_list)
+    # More informative assertion
+    real_errors = [err for err in errors_list if err is not None]
```

Comment:
the errors should never be `None`. That happened because a combination of two things:
- the new errors that were introduced in the previous PR
- and an error in the test: instead using `print_return_msg` there is `print` function added to the error list in one place, so there are `None`s instead of the message itself. I fixed it in #86, see [here](https://github.com/ReproNim/reproschema-py/blob/92e34b9007e9074de4be0d5ab19697053f3d0111/reproschema/tests/test_rs2redcap_redcap2rs.py#L224)

So unfortunately it means that there are still errors